cloudformation/functions/tests/test_get_groups_from_policy.py

```diff
@@ -0,0 +1,42 @@
+import json
+import os
+import os.path
+import sys
+from cloudformation.functions.get_groups_from_policy import (
```

Comment:
Interesting. I was doing relative imports in mine like `from ..get_groups_from_policy import get_groups_from_policy`. Should I switch to this absolute form or should we switch this to relative (I don't know what's best for unit tests)

Reply:
Relative imports are fine most of the time, but there are issues with them when run not in a module, e.g. executed directly. You'll raise `ValueError: Attempted relative import in non-package`.

In general, I prefer absolute imports as they are a lot clearer, except when dealing with very large programs with complex import paths. Not a huge deal either way.